CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.3.4 (2020-03-20)
+
+- Fixed the country field component to properly update when not initialized with a value within a reactive form. [#71](https://github.com/blackbaud/skyux-lookup/pull/71)
```

Comment:
Fixed the country field component to update without initialization from values within reactive forms.

Does my suggestion change the meaning?

Reply:
Ya... I don't think that works in this case.

Reply:
How about `Fixed the country field component to properly handle the first value change on a reactive form when the initial value is undefined.`?

Reply:
That works. Updated